src/meta_memcache/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "1.1.2"
+__version__ = "2.0.1"
```

Comment:
Is there any way to get uv to inject this somehow? It seems easy to forget to change it.